app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/services/MultipleSelfEmploymentsService.scala

```diff
@@ -38,7 +38,7 @@ class MultipleSelfEmploymentsService @Inject()(incomeTaxSubscriptionConnector: I
 
   def saveBusinessStartDate(businessId: String, businessStartDate: BusinessStartDate)
                            (implicit hc: HeaderCarrier): Future[Either[SaveSelfEmploymentDataFailure.type, PostSelfEmploymentsSuccess]] = {
-    saveData(businessId, _.copy(businessStartDate = Some(businessStartDate)))
+    saveData(businessId, _.copy(businessStartDate = Some(businessStartDate), confirmed = false))
```

Comment:
hi Samir, can I know if those confirmed = false is necessary when it already defaults to false in the SelfEmploymentData model?

Reply:
The `confirmed` flag gets set to true when the user confirms the data via the CYA page. It's necessary to reset the flag when users update any field.

Reply:
I see, thanks